README.md

```diff
@@ -160,8 +178,8 @@ If you believe you have found a security vulnerability in this project or any of
 
 If you would like to contribute to this project, review [these guidelines](./CONTRIBUTING.md).
 
-To all contributors, we thank you!  Without your contribution, this project would not be what it is today.  We also host a community project page dedicated to [Project Name](<LINK TO https://opensource.newrelic.com/projects/... PAGE>).
+To all contributors, we thank you!  Without your contribution, this project would not be what it is today.  We also host a community project page dedicated to Newrelic Prometheus Configurator(<LINK TO https://opensource.newrelic.com/projects/... PAGE>).
```

Comment:
NIT: Could you add a `TODO`? To make it grepable or allow the IDE to follow them.

Reply:
We still need to change `<LINK TO https://opensource.newrelic.com/projects/... PAGE>` So simple add a `TODO` at the end of the line to mark it as TODO.

A stupid NIT, you can completely ignore it.

Reply:
I'll add it in one of the ga tasks , thanks